src/test/resources/arimax_data.csv

```diff
@@ -0,0 +1,258 @@
+38.3333
```

Comment:
Do they already have a pattern of added csv files into test/resources?  Where did this data come from?  just need to be careful about any licensing...